### PR TITLE
Removing unused static pages; refactor existing tests

### PR DESCRIPTION
**Pull request by awead**
This addresses #728, but as far as I can tell, the videos aren't there anymore. However, there are some other things that need cleaning up. There are some static pages that aren't used anymore: error_pages and subject_libraries. I can find no links to these pages and they aren't tested anywhere.

I've also edited the help page to remove the internal anchor links. The page is too short for them to be effective. I've also removed all of the commented-out html code.

The feature tests have been updated to reflect the existing pages.

### CODE REVIEW

File: app/views/static/help.html.erb [hunk 1]
```diff
@@ -1,21 +1,9 @@
 <% provide :page_title, "Help | ScholarSphere" %>
 <h1 class="sr-only">Help</h1>
 
-<h2> Table of Contents</h2>
```
Comment (carolyncole) [inline app/views/static/help.html.erb]: @awead This was added for accessibility.  We still have other sections so removing this make the page less accessible.

File: app/views/static/subject_libraries.html.erb [hunk 2]
```diff
@@ -1,35 +0,0 @@
-<h1> Subject Libraries </h1>
```
Comment (carolyncole) [inline app/views/static/subject_libraries.html.erb:1]: @awead Have you looked at google scholar to see if this page is accessed?

**Review by carolyncole — CHANGES_REQUESTED** [review]
Looks like you are removing an accessibility item.

File: app/views/static/subject_libraries.html.erb (hunk 2 above)
Comment (awead) [reply app/views/static/subject_libraries.html.erb:1]: No, how do I do that?

File: app/views/static/help.html.erb (hunk 1 above)
Comment (awead) [reply app/views/static/help.html.erb]: Could we hide it then? It doesn't do anything for sighted users, other than reload the page.

Comment (awead) [reply app/views/static/help.html.erb]: @cam156 I thought the headings themselves outlined the structure, which is what we're doing with the other pages, ensuring that headings move from H1, H2, etc. With the screen reader on, I can get an overview of the page just from the headings:
help

File: app/views/static/subject_libraries.html.erb (hunk 2 above)
Comment (awead) [reply app/views/static/subject_libraries.html.erb:1]: @cam156 looks like it was removed at some point in QA: https://scholarsphere-qa.dlt.psu.edu/subject_libraries
It's still working in prod: https://scholarsphere.psu.edu/subject_libraries

**Comment by awead** [comment]
@cam156 We agreed that subject_libraries could be removed. I put the TOC back into the help page.  Thanks!

**Review by carolyncole — APPROVED** [review]

**Pull request merged**